project.php

```diff
@@ -1401,6 +1401,7 @@ function do_post_files()
         echo "<ul>";
 
         echo_download_zip(_("Download Zipped Images"), 'images');
+        echo_download_zip(_("Download Zipped Music Files"), '_music');
```

Comment:
why `_music`? Also, could we make it a constant since it is referenced in more than one place? It'd make updating it later for any reason easier.

Reply:
We could do that. I suppose we should really have a constant for `image` also. I'll leave it like it is for now.